### ISSUE TITLE
Sign in doesn't work, opens a blank pop up window

### ISSUE THREAD

**Issue by TigerNightmare** [Seldszar/Gumbo#5]
Tried to use this to replace Twitch Now, but whenever I click on the "Login with Twitch" button, a new window briefly pops up with no address in the URL bar, then disappears. Nothing new shows up in my history, either. I tried disabling pop up blocking and several other extensions, but nothing seems to make a difference. Any suggestions?

![image](https://user-images.githubusercontent.com/5782645/156313374-2e0af133-381b-4abe-b362-6a4a389540ff.png)

I installed the version from the Firefox Add-ons site.

**Comment by Seldszar** [comment]
What version of Firefox are you using?

**Comment by TigerNightmare** [comment]
I'm using 97.0.1 64-bit.

**Comment by Seldszar** [comment]
Is it possible for you to try with a fresh or portable version of Firefox and see if there's still the issue?

**Comment by TigerNightmare** [comment]
I installed a portable Firefox, and as soon as I signed in, it installed all of my extensions again and I had the same problem, so I installed a separate one and didn't sign in, which worked. I went back to the regular desktop Firefox and disabled everything until it worked. It turns out that my extension called Conex, which is for creating and managing tab groups, was causing the issue. I don't know why, it's never caused issues with similar authentication windows. Thanks for the help, much appreciated.
